plugin/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "kubectl-openebs"
+version = "0.7.0"
+edition = "2021"
+
+[[bin]]
+name = "kubectl-openebs"
+path = "src/main.rs"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+kubectl-plugin = { path = "../mayastor/k8s/plugin", default-features = false }
+rest-plugin = { path = "../mayastor/dependencies/control-plane/control-plane/plugin", default-features = false }
+shutdown = { path = "../mayastor/dependencies/control-plane/utils/shutdown"}
+utils = { path = "../mayastor/dependencies/control-plane/utils/utils-lib" }
+upgrade = { path = "../mayastor/k8s/upgrade" }
+anyhow = "1.0.75"
+async-trait = "0.1.73"
+clap = { version = "4.4.6", features = ["color", "derive"] }
+k8s-openapi = { version = "0.20.0", features = ["v1_22"] }
```

Comment:
these are no longer up to date with mayastor-extensions

Reply:
Ill change it next PR.